internal/cli/atlas/integrations/create/new_relic.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const newRelicIntegrationType = "NEW_RELIC"
+
+type NewRelicOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	licenseKey string
+	accountID  string
+	writeToken string
+	readToken  string
+	store      store.IntegrationCreator
+}
+
+func (opts *NewRelicOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var createTemplate = "New Relic integration created.\n"
```

Comment:
depending on my previous question, if you can update using the create store then we should reword this, maybe something like `"New Relic integration configured.\n" or something

Reply:
@andreaangiolillo may want to take a look at this then